src/common/components/PageLayout.jsx

```diff
@@ -82,10 +93,17 @@ const PageLayout = ({ menu, breadcrumbs, children }) => {
         classes={{ paper: classes.desktopDrawer }}
       >
         <Toolbar>
-          <IconButton color="inherit" edge="start" sx={{ mr: 2 }} onClick={() => navigate('/')}>
-            <ArrowBackIcon />
+          {!miniVariant && (
+          <>
+            <IconButton color="inherit" edge="start" sx={{ mr: 2 }} onClick={() => navigate('/')}>
+              <ArrowBackIcon />
+            </IconButton>
+            <PageTitle breadcrumbs={breadcrumbs} />
+          </>
+          )}
+          <IconButton color="inherit" edge="start" sx={{ ml: miniVariant ? -2 : 'auto' }} onClick={toggleDrawer}>
```

Comment:
I think this needs to be reformatted.

Reply:
which part?

Reply:
got it

Reply:
I think we should probably apply classes here as well for readability.

Reply:
I followed the same pattern as the upper IconButton